R/test-server.R

```diff
@@ -55,75 +58,78 @@ isModuleServer <- function(x) {
 #'   # Any additional arguments, below, are passed along to the module.
 #' })
 #' @export
-testServer <- function(app = NULL, expr, args = list(), session = MockShinySession$new()) {
+testServer <- function(app, expr, args = list(), session = MockShinySession$new()) {
```

Comment:
We had it default to `NULL` so that in the case where you're testing an application on disk with `runTests()`, it just tests the server function from the current app. This is done in the app template (ignore the stuff with `{{`, since that's templating code): https://github.com/rstudio/shiny/blob/master/inst/app_template/tests/testthat/test-server.R#L3 

Does this now require users to explicitly use `app=NULL` if they want to test the current app?

Reply:
Yes, it would. I'm happy to change the default back, but I personally think the most common use of this function will be in regular unit test files 😄